android/src/main/java/com/swmansion/gesturehandler/core/GestureHandlerOrchestrator.kt

```diff
@@ -85,11 +85,18 @@ class GestureHandlerOrchestrator(
   private fun hasOtherHandlerToWaitFor(handler: GestureHandler<*>) =
     gestureHandlers.any { !isFinished(it.state) && shouldHandlerWaitForOther(handler, it) }
 
-  private fun shouldBeCancelledByFinishedHandler(handler: GestureHandler<*>) = gestureHandlers.any { shouldHandlerWaitForOther(handler, it) && it.state == GestureHandler.STATE_END }
+  private fun shouldBeCancelledByFinishedHandler(handler: GestureHandler<*>) =
+    gestureHandlers.any { shouldHandlerWaitForOther(handler, it) && it.state == GestureHandler.STATE_END }
+
+  private fun handlersWithinHandlerBounds(handler: GestureHandler<*>) =
```

Comment:
This name is quite misleading. For me it suggests that we compare positions of native views, while in reality we check for common pointers.

What about `handlersWithCommonPointers`?

Reply:
this function was removed in [6d11f86](https://github.com/software-mansion/react-native-gesture-handler/pull/3095/commits/6d11f86f2db7aa9985793febffdeddea9c8e1c26)